### ISSUE TITLE
Add a LICENSE

### ISSUE THREAD

**Issue by kalakris** [richo/homeassistant-franklinwh#18]
I assume the intention was to have this package be open source, but it would be good to add a license to this repo to remove this ambiguity and allow others to contribute.

**Comment by richo** [comment]
Good call. It's dual Apache/mit but I'll add a license tonight. Thankyou!
